ani-cli

```diff
@@ -376,16 +376,19 @@ tput cuu1 && tput el
 play
 [ "$player_function" = "download" ] || [ "$player_function" = "debug" ] && exit 0
 
-while cmd=$(printf "quit\nselect\nprevious\nreplay\nnext" | nth "Playing episode $ep_no of $title... "); do
+while cmd=$(printf "quit\nselect\nprevious\nreplay\nnext\nchange_quality" | nth "Playing episode $ep_no of $title... "); do
 	case "$cmd" in
 	previous) ep_no=$(printf "%s" "$ep_list" | sed -n "/^${ep_no}$/{n;p;}") 2>/dev/null ;;
 	replay) ;;
 	next) ep_no=$(printf "%s" "$ep_list" | sed -n "/^${ep_no}$/{g;1!p;};h") 2>/dev/null ;;
 	select) ep_no=$(printf "%s" "$ep_list" | nth "Select episode: " "$multi_selection_flag") ;;
+	change_quality) episode=$(printf "%s" "$links" | sed -n '/^\([0-9]*p\)/p' | launcher)
+	episode=$(printf "%s" "$episode" | cut -d'>' -f2);;
 	*) exit 0 ;;
 	esac
 	[ -z "$ep_no" ] && die "Out of range"
 	play
+	episode=
```

Comment:
this looks a bit sus

Reply:
dont worry its to adjust with 214, selecting quality in the menu just uses already scraped link so when you press next without emptying episode it'll just replay the same episode again and again

tested everything as well , should be all fine